code/game/turfs/turf.dm

```diff
@@ -48,7 +48,7 @@ GLOBAL_LIST_EMPTY(station_turfs)
 	///List of light sources affecting this turf.
 	var/tmp/list/datum/light_source/affecting_lights
 	///Our lighting object.
-	var/tmp/atom/movable/lighting_object/lighting_object
+	var/tmp/datum/lighting_object/lighting_object
```

Comment:
Why is this using tmp, we don't save objects anywhere.

Reply:
no idea. dont know what tmp does

Reply:
It prevents variables from being saved when dumped to savefiles directly. We do not use that method anywhere.

Reply:
ok tmp is in the reference despite static isnt, yeah i dont see why it uses it

Reply:
this system came from vg who got it from goon. likely one of them used it.